packages/core-components/src/components/multiselect-dropdown/multiselect-dropdown.tsx

```diff
@@ -69,6 +73,19 @@ export class B2bMultiSelectDropdown {
     this.b2bChange.emit(newValues);
   }
 
+  private parseProps() {
```

Comment:
The method name parseProps seems generic. Can we name it restricted to option related props like parseOptionProps?